File: Hivemind.Ng/src/autogenerated/entities/InjuryReport.ts
import { GangerInjuryReport } from './GangerInjuryReport';

export class InjuryReport {
    public injuries: GangerInjuryReport[];

    public constructor(partial: Partial<InjuryReport>) {
        if (partial.injuries) {
            this.injuries = partial.injuries;
        }
    }
}

File: Hivemind.Api.Core/Contracts/InjuryReport.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hivemind.Contracts
{
    public class InjuryReport
    {
        public IEnumerable<GangerInjuryReport> Injuries { get; set; }
    }
}
